tarorpc/taro.proto

```diff
@@ -285,6 +290,53 @@ message ListBalancesResponse {
     map<string, AssetFamilyBalance> asset_family_balances = 2;
 }
 
+message ListTransfersRequest {
+}
+
+message ListTransfersResponse {
+    // The unordered list of outgoing asset transfers.
+    repeated AssetTransfer transfers = 1;
+}
+
+message AssetTransfer {
+    int64 transfer_timestamp = 1;
+
+    // The old/current location of the Taro commitment that was spent as an
+    // input.
+    string old_anchor_point = 2;
+
+    // The new location of the Taro commitment referenced by the old anchor
+    // point.
+    string new_anchor_point = 3;
+
+    // The new Taro root that commits to the set of modified and unmodified
+    // assets.
+    bytes taro_root = 4;
```

Comment:
Totally non-blocking, but I'm thinking we also want to start showing the branch hash and also value here. Then people can spot check the roots to see things like the total number of units, etc.